app/helpers/accounts_helper.rb

```diff
@@ -21,6 +21,14 @@ def account_fullname_with_matriculation_number(account)
     "#{account.fullname} (#{account.matriculation_number})"
   end
 
+  def account_forname(account)
+    "#{account.forename}"
+  end
+
+  def account_surname(account)
+    "#{account.surname}"
+  end
+
```

Comment:
I think these helper methods are not necessary, since they do not provide any benefits over just calling `account.forename` and `account.surname` respectively
```suggestion
```